src/main/java/com/amihaiemil/docker/Container.java

```diff
@@ -65,4 +65,12 @@
      *     expected.
      */
     void stop() throws IOException, UnexpectedResponseException;
+
+    /**
+     * Restarts the container.
+     * @throws IOException If something goes wrong.
+     * @throws UnexpectedResponseException If the status response is not
+     *     expected.
+     */
+    void restart() throws IOException, UnexpectedResponseException;
```

Comment:
@llorllale UnexpectedResponseException is runtime exception, no need to declare it :D

Reply:
@amihaiemil checkstyle fails if we don't include it.

Reply:
@llorllale strange, I don't remember checkstyle being that strict. Do you know how to add an exception for it quickly, or should we merge and I'll look into it later?

Reply:
@amihaiemil to be clear, checkstyle fails when I add the `@throws` tag without declaring the exception. What you want is to include `@throws` *without* declaring the exception in `throws`, correct?

Reply:
@llorllale you're right, let's leave it this way then.